doc/man7/provider-base.pod

```diff
@@ -405,6 +406,25 @@ The number of bits of security offered by keys in this group. The number of bits
 should be comparable with the ones given in table 2 and 3 of the NIST SP800-57
 document.
 
+=item "tls-group-is-kem" (B<OSSL_CAPABILITY_TLS_GROUP_IS_KEM>) <unsigned integer>
+
+Boolean flag to describe is the group should be used in KEX mode (0, default) or
+in KEM mode (1).
+
+This parameter is optional: if not specified, KEX mode is assumed as the default
+mode for the group.
+
+In KEX mode, in a typical Diffie-Hellman fashion, both sides execute I<keygen>
+then I<derive> against the peer public key.
```

Comment:
Most importantly it needs to support the provider functions as described in provider-keyexch.pod.

Reply:
Fixed in upcoming commit.